index.html

```diff
@@ -3,134 +3,477 @@
+    .bar text {
+      fill: #fff;
+      font: 10px sans-serif;
     }
+
+    .ybar {
+      stroke: rgb(240, 59, 32);
+      stroke-width: 1;
+      fill: rgb(240, 59, 32);
+      opacity: 1;    }
+
+    .ybar text {
+      fill: #fff;
+      font: 10px sans-serif;
+    }
+
+   
+    .axis text {
+        font-family: 'Arimo', sans-serif;
+        font-size: 11px;
+        fill: #CECECE;
+      }
+
+    label {
+      font-weight: 2px;
+      color: #FFFFFF;
+    }
+
+    body {
+      background-color: #4C4C4C;
+      font-family: 'Arimo', sans-serif;
+      color: #CECECE;
+          }
+
+    th {
+       font-family: 'Exo', sans-serif;
+        color: #FFFFFF;
+        width: 400px;
+                } 
+    h1,h3,h5,h2{
+        font-family: 'Exo', sans-serif;
+        color: #FFFFFF;
+                }   
+    #year-range,#region-select {
+      background-color: #CECECE;
+      color: #4C4C4C;
+      border: none;
+      border-radius: 24px;
+
+    .axis path,
+    .axis line {
+      fill: none;
+      stroke: #CECECE;
+      shape-rendering: crispEdges;
+                }
+
+    .btn-toolbar {
+        font-weight: 2px;
+      color: #FFFFFF;
+    }
+    .tooltip {
+       padding: .5em 1em;
+       background: rgba(0,0,0,.75);
+       color: white;
+       font-size: 13px;
+       font-weight: 700;
+       line-height: 1;
+     }
+
+    
   </style>
 </head>
 <body>
 <div class="header section">
   <div class="container">
-    <h1>Week 4: Replication Project</h1>
-    <p>Let's build a complex interactive graphic.</p>
+    <h1>Poverty and Crime - United States</h1>
+    <p>Counties in the United States, sized by population, where data is available. Crime rate is total arrests per 100,000 people in 2012. <br>
+      Sources: US Census Bureau and UMich National Archive of Criminal Justice Data</p>
   </div>
 </div>
 <div class="controls section">
   <div class="container">
     <div class="btn-toolbar">
-      <label for="year-range">Year range: </label>
-      <select id="year-range">
-        <option value="all">All (1948&ndash;2016)</option>
-        <option value="1955-1971">1955&ndash;1971</option>
-        <option value="1974-1984">1974&ndash;1984</option>
-        <option value="1985-1992">1985&ndash;1992</option>
-        <option value="1997-2007">1997&ndash;2007</option>
-        <option value="2008-2016">2008&ndash;2016</option>
+      <label for="region-select">Region: </label>
+      <select id="region-select">
+        <option value="all">All</option>
+        <option class="select-dash" disabled="disabled">-Region-</option>
+        <option value="Northeast">Northeast</option>
+        <option value="Southeast">Southeast</option>
+        <option value="West">West</option>
+        <option value="Midwest">Midwest</option>
+        <option value="Southwest">Southwest</option>
       </select>
+      <label for="year-range">State: </label>
+      <select id="year-range">
+        <option value="all">All</option>
```

Comment:
year-range isn't a particularly intuitive name for this form, hah. Might be worth changing - just remember to do it in the selectors too.